### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by petroav**
This brings test coverage to 88%.

Could someone also edit the description of the project in Github? I think this repo no longer contains `Test utilities for spinning up swarm networks`. In the README the description I've used is `Utility methods for creating public and private keys for use in tests`; feel free to use that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+go-libp2p-netutil
+==================
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
```
Comment (Stebalien) [inline README.md]: This should be `https://protocol.ai`.

Comment (petroav) [reply README.md]: Done.

**Comment by petroav** [comment]
Hey @Stebalien -- thanks for taking a look! Just updated branch.

**Comment by Stebalien** [comment]
Thanks! Better test coverage and docs are always welcome!

**Pull request merged**